test/integration/cli/test_launch.py

```diff
@@ -495,3 +495,47 @@ def test_std_err_stream_local_launch_non_existent_dir(tt_cmd, tmpdir):
     assert tt_process.returncode == 1
     assert "failed to change working directory" not in tt_process.stdout.readline()
     assert "failed to change working directory" in tt_process.stderr.readline()
+
+
+def test_launch_with_env_cfg(tt_cmd, tmpdir):
+    cmd = [tt_cmd, "cfg", "dump"]
+    # Create cfg in separate directory and set env.
+    tmpdir_with_env_config = tempfile.mkdtemp()
+    create_tt_config(tmpdir_with_env_config, tmpdir)
+    test_env = os.environ.copy()
+    test_env['TT_CLI_CFG'] = tmpdir_with_env_config + "/tt.yaml"
+
+    rc, output = run_command_and_get_output(cmd, cwd=os.getcwd(), env=test_env)
+    assert rc == 0
+    assert tmpdir_with_env_config + "/tt.yaml" in output
```

Comment:
We can write anything to the config and check the `dump` output.

Reply:
changed check